plugins/celery_jobs/web_client/templates/configView.pug

```diff
@@ -1,16 +1,16 @@
 .g-config-breadcrumb-container
 form#g-celery-jobs-config-form(role="form")
   .form-group
-    label.control-label(for="celery_jobs_broker") Broker URL
-    input.input-sm.form-control#celery_jobs_broker(
+    label.control-label(for="g-celery-jobs-broker") Broker URL
```

Comment:
I might otherwise complain here, but this whole plugin is on the chopping block :)